php-practice/section7.php

```diff
@@ -76,3 +95,15 @@
  *
  * (It returns TRUE if $needle is a substring of $haystack; FALSE otherwise.)
  */
+
+function stringContains($haystack, $needle) {
+	$pos = strpos($haystack, $needle);
+
+		if ($pos == true) {
+			return true;
+		} else {
+			return false;
+		}
+}
```

Comment:
So I was wrong about my original analysis of this function. It will *not* always return the correct value.

```PHP
stringContains('donuts', 'nuts'); // Returns true correctly; strpos will return 2 because that is the position of the substring "nuts" in "donuts"
stringContains('donuts', 'don'); // Returns false incorrectly; strpos will return 0 because that is the position of the substring "don" in "donuts". 0 loosely evaluates to false, not true.